thunder/tests/test_inplace_copy.py

```diff
@@ -125,21 +125,21 @@ def func0(x, y):
 
     def func1(x, y):
         z = x * y
-        thunder.core.prims.copy_(z, x)
-        thunder.core.prims.copy_(y, x)
-        return x
+        o1 = thunder.core.prims.copy_(z, x)
+        o2 = thunder.core.prims.copy_(y, x)
+        return x, o1, o2
```

Comment:
These `copy_`'s bypasses the name swapping in functionalization, so we have to manually return their outputs so that they will be ordered after `copy_`.